templates/ords-remix-jwt-sample/app/root.tsx

```diff
@@ -42,7 +41,7 @@ export const loader = async ({ request }: LoaderFunctionArgs) => {
   const userProfile = await auth.isAuthenticated(request);
   const profile = userProfile?.profile || null;
   const USER_CREDENTIALS = userProfile === null
-    ? BASIC_SCHEMA_AUTH
+    ? ''
```

Comment:
Is there a reason why the user credentials are left empty if there is no user profile? 
I see that we pass this variable to the `ORDSFetcher` function (https://github.com/oracle/create-database-app/blob/main/templates/ords-remix-jwt-sample/app/utils/ORDSFetcher.ts). I think we should omit the `Authorization` header if the `USER_CREDENTIALS` variable is empty (see https://github.com/oracle/create-database-app/blob/597e88ed6454f6450b7a231ce490b22af36140cb/templates/ords-remix-jwt-sample/app/utils/ORDSFetcher.ts#L41).

Reply:
Also, the `USER_CREDENTIALS` parameter should be optional, this way we make it clear that the queried endpoints are public.